mkdocs/contrib/search/templates/search/lunr.js

```diff
@@ -0,0 +1,2977 @@
+/**
+ * lunr - http://lunrjs.com - A bit like Solr, but much smaller and not as bright - 2.1.5
```

Comment:
FYI Lunr [2.1.6](https://github.com/olivernn/lunr.js/blob/master/CHANGELOG.md#216) was released which includes a performance improvement when building indexes. I don't know how you intend on keeping up with releases of Lunr, but thought you might appreciate faster building!

Reply:
Thanks for the heads up. Might as well release with the most recent version.